client/src/component/MilestoneTable.js

```diff
@@ -0,0 +1,10 @@
+export function MilestoneTable() {
+
+	return (
+			<div>
+				<h1>hello world</h1>
+			</div>
+	);
+}
+
+export default MilestoneTable;
```

Comment:
It would be better to export a  component once, at the first component or at the last.